e2e-tests/portal/cypress/support/commands.js

```diff
@@ -83,6 +83,7 @@ Cypress.Commands.add('renameDeal', require('./portal/renameDeal'));
 Cypress.Commands.add('resetPortalUserStatusAndNumberOfSignInLinks', require('./portal/resetPortalUserStatusAndNumberOfSignInLinks'));
 Cypress.Commands.add('userSetPassword', require('./portal/userSetPassword'));
 Cypress.Commands.add('disablePortalUserByUsername', require('./portal/disablePortalUserByUsername'));
+Cypress.Commands.add('createBssApplicationReadyForCheck', require('./portal/createBssDealReadyForCheck'));
```

Comment:
Should be `createBssDeal` instead of `createBssApplicationReadyForCheck` please

Reply:
Changed to `createBssDeal`

Reply:
I was wondering if it should be `createBssEwcsDeal`?

Reply:
Resolving, has been renamed.